minecraft_servers/mush/manifest.json

```diff
@@ -4,7 +4,8 @@
   "direct_ip": "jogar.mush.com.br",
   "server_wildcards": [
     "%.mush.com.br",
```

Comment:
Is not a real wildcard
https://i.imgur.com/oUoCLHY.png

Reply:
Wildcard should be working now.